dataland-backend/src/main/kotlin/org/dataland/datalandbackend/configurations/SwaggerUiConfig.kt

```diff
@@ -0,0 +1,23 @@
+package org.dataland.datalandbackend.configurations
+
+import org.springdoc.core.models.GroupedOpenApi
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+/**
+ * This class is used to configure the CSP for the Swagger-UI as the default content does not allow loading anything
```

Comment:
Was bedeutet der Satz?

Reply:
Nothing that would make sense. I corrected it.

Reply:
ok